### PR TITLE
Add: Lazygit

### PR DESCRIPTION
**Pull request by kcubeterm**
#5884

### CODE REVIEW

**Comment by kcubeterm** [comment]
Is it necessary to use `install`.

**Comment by Grimler91** [comment]
> Is it necessary to use `install`.

You mean the `install -Dm700 ${TERMUX_PKG_BUILDDIR}/src/github.com/jesseduffield/lazygit/lazygit $TERMUX_PREFIX/bin/lazygit` command? 
It is very convenient to use it, it is the same as running:
```
mkdir -p $TERMUX_PREFIX/bin
cp ${TERMUX_PKG_BUILDDIR}/src/github.com/jesseduffield/lazygit/lazygit $TERMUX_PREFIX/bin/
chmod 700 $TERMUX_PREFIX/bin/lazygit
```
i.e. it makes sure the install folder exists and that the installed file has the right permissions (rwx in this case)

File: packages/lazygit/build.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jesseduffield/lazygit
+TERMUX_PKG_DESCRIPTION="simple terminal UI for git commands"
```
Comment (Grimler91) [inline packages/lazygit/build.sh:2]: Please change to "Simple" (capital S)

**Comment by kcubeterm** [comment]
> You mean the install -Dm700 ${TERMUX_PKG_BUILDDIR}/src/github.com/jesseduffield/lazygit/lazygit

No, not this one. 

```sh
 cp -a ${TERMUX_PKG_BUILDDIR}/src/github.com/jesseduffield/lazygit/docs/* \
                $TERMUX_PREFIX/share/doc/lazygit/

```

This, one

**Comment by Grimler91** [comment]
> ```shell
>  cp -a ${TERMUX_PKG_BUILDDIR}/src/github.com/jesseduffield/lazygit/docs/* \
>                 $TERMUX_PREFIX/share/doc/lazygit/
> ```
> 
> 
> This, one

Docs are nice I guess, not familiar with lazygit so can't tell how useful in this particular package are though

**Comment by kcubeterm** [comment]
Yea, Docs are nice. 

BTW. I got my answer. thanks.

**Review by Grimler91 — APPROVED** [review]

**Pull request merged**